SpeechResponder/HelpWindow.xaml.cs

```diff
@@ -17,17 +17,19 @@ public HelpWindow()
 
             // Read Markdown and convert it to HTML
             string markdown;
+            string blabla = "";
             try
             {
                 DirectoryInfo dir = new DirectoryInfo(System.IO.Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location));
-                markdown = Files.Read(dir.FullName + @"\Help.md");
+                markdown = File.ReadAllText(dir.FullName + @"\Help.md"); //Files.Read(dir.FullName + @"\Help.md");
```

Comment:
This is what I'd have done in the first place. Not sure where Files.Read() comes from, can anyone enlighten me?

Reply:
It's one of JGM's commits... commit #b1b212c8 on May 7th of this year. 
The commit changed the line from File.ReadAllText to Files.Read.
The commit description reads "Tidy up file reading and writing".

Reply:
Ahh, I think I found it. 
In Files.cs, JGM defines the 'Files' class and the 'Read' method. It does a try with File.ReadAllText and catches exceptions. Consequently, I don't think this change should be necessary.